msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -37,44 +37,64 @@ Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#
+| [**selectMedia**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest&branch=master#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true) for image capturing in camera| This API allows users to **capture or select media from the device camera** and return it to the web-app. The users can edit, crop, rotate, annotate, or draw over images before submission. In response to **selectMedia**, the web-app receives the media IDs of selected images and a thumbnail of the selected media.|
+| [**selectMedia**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest&branch=master#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true) for audio recording in **Microphone**| Set the media type to `four` in **selectMedia** API for accessing microphone  capability. This API also allows users to record audio from the device microphone and return recorded clips to the web-app. The users can pause, re-record, and play recording preview before submission. In response to **selectMedia**, the web-app receives media IDs of the selected audio recording. <br/> Use `maxDuration`, if you require to configure a duration in minutes for recording the conversation. The current duration for recording is 10 minutes, after which the recording terminates.  |
+| [**getMedia**](/javascript/api/@microsoft/teams-js/_media?view=msteams-client-js-latest&branch=master#getMedia__error__SdkError__blob__Blob_____void_&preserve-view=true)| This API retrieves the media captured by **selectMedia** API in chunks, irrespective of the media size. These chunks are assembled and sent back to the web app as a file or blob. Breaking of media into smaller chunks facilitates large file transfer. |
+| [**viewImages**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#viewImages_ImageUri_____error___SdkError_____void_&preserve-view=true)| This API enables the user to view images in  full-screen mode as a scrollable list.|
 
-**Web app experience for selectMedia API**
-![device camera and image experience in Teams](../../assets/images/tabs/image-capability-screenshot.jpg)
+
+**Web app experience for selectMedia API for image capability**
+![device camera and image experience in Teams](../../assets/images/tabs/image-capability.png)
+
+**Web app experience for selectMedia API for microphone capability**
+![web app experience for microphone capability](../../assets/images/tabs/microphone-capability.png)
 
 ## Error handling
 
-You should understand the API response error codes and handle them appropriately. Below is a list of error codes that may be returned by the platform:
+The following table lists the error codes and the conditions under which the errors are generated. You must ensure to handle these errors appropriately in your Teams app.
```

Comment:
Since you are using following, please add a colon instead of a period at the end of the sentence.